src/App.js

```diff
@@ -1,21 +1,25 @@
+/* eslint-disable import/extensions */
+import { useSelector } from 'react-redux';
 import React from 'react';
-import {
-  Routes,
-  Route,
-} from 'react-router-dom';
-import NavBar from './components/navbar';
-import Categories from './components/categories';
-import Books from './components/books';
+import { Route, Routes } from 'react-router-dom';
+import './index.css';
+import Books from './components/Books';
+import Navbar from './components/Navbar';
+import Categories from './components/Categories';
```

Comment:
Your project looks great. However, I can't run the project successfully.

This is because you have imported `./components/Books`, `./components/Navbar` and `./components/Categories` with different names from what they're saved with which are `./components/books`, `./components/navbar` and `./components/categories` respectively.